### PR TITLE
refactor all the things

### PR DESCRIPTION
**Pull request by markparolisi**

### CODE REVIEW

**Comment by markparolisi** [comment]
@matstars 
The behavior should still be the same. Just rewrote a bunch of things

Tested on WP 3.9

File: voce-post-meta-widgets.php [hunk 1]
```diff
@@ -161,166 +255,97 @@ public static function plugins_url( $relative_path, $plugin_path ) {
...
-			<?php wp_list_widgets(); ?>
-		</div>
-		<?php wp_nonce_field( 'save-sidebar-widgets', '_wpnonce_widgets', false ); ?>
-		<div class="clear"></div>
-		<?php
-	}	
 
 	/**
 	 *
 	 * Clean sidebars, removes any sidebars that are empty
```
Comment (matgargano) [inline voce-post-meta-widgets.php:261]: should we array_unique the sidebars as well? Seems like I'm getting duplicates in `get_option('voce_post_meta_sidebars');``

File: README.md [hunk 2]
```diff
@@ -50,8 +50,11 @@ add_filter('voce_post_meta_widgets_post_types', function( $current_post_types )
 ?>
 ```
 
-**0.1.1**  
+**0.1.2**
+* Refactored load order
+
+**0.1.1**
 *Update to check for Voce_Meta_API*
```
Comment (matgargano) [inline README.md]: clean these up to be consistent with 0.1.2

File: README.md [hunk 3]
```diff
@@ -2,11 +2,11 @@ Voce Post Meta Widgets
 ==================
 
 Contributors: matstars, voceplatforms
-Tags: post, meta, widgets, widget area  
-Requires at least: 3.3  
-Tested up to: 3.8.1  
-Stable tag: 0.1.1  
-License: GPLv2 or later  
+Tags: post, meta, widgets, widget area
```
Comment (matgargano) [inline README.md]: header of readme.md not formatted properly

File: README.md [hunk 4]
```diff
@@ -2,11 +2,11 @@ Voce Post Meta Widgets
...
+Requires at least: 3.3
+Tested up to: 3.9.0
+Stable tag: 0.1.2
+License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
```
Comment (matgargano) [inline README.md]: Can you add in an example of how to retrieve & output the sidebars (i.e. when writing a front-end template) to the readme?

File: README.md (hunk 2 above)
Comment (markparolisi) [inline README.md]: Will do. Not sure why there is a readme.md and a readme.txt though. The .txt should suffice for both WP.org and GH

Comment (matgargano) [inline README.md]: feel free to get rid of the .md

**Comment by markparolisi** [comment]
@matstars Can this get reviewed and merged today?

File: README.md (hunk 4 above)
Comment (matgargano) [inline README.md]: @markparolisi I think this is the last outstanding item

**Pull request merged**